File: pinyougou-parent/pinyougou-manager-web/src/main/webapp/js/service/brandService.js
app.service('brandService', function ($http) {
    this.search = function (page, pageSize, searchEntity) {
        return $http.post('../brand/findPage.do?page='+page+'&pageSize='+pageSize,searchEntity);
    };

    this.findOne = function (id) {
        return $http.get('../brand/findOne.do?id='+id);
    };

    this.delete = function (selectIds) {
        return  $http.get("../brand/delete.do?ids="+selectIds);
    };

    this.add = function (entity) {
        return $http.post('../brand/add.do',entity)
    };

    this.update = function (entity) {
        return $http.post('../brand/update.do',entity)
    };

    // 下拉表数据
    this.selectOptionList=function(){
        return $http.get('../brand/selectOptionList.do');
    };
});

File: pinyougou-parent/pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.pojo.TbBrand;
import entity.PageResult;

import java.util.List;
import java.util.Map;

public interface BrandService {
	List<TbBrand> findAll();

	PageResult findPage(TbBrand brand, Integer page, Integer pageSize);

	void add(TbBrand brand);

	void update(TbBrand brand);

	TbBrand findOne(Long id);

	boolean isNameInBrandNames(String name);

	void deleteByIds(Long[] ids);

	List<Map> selectOptionList();
}
